docs/quickstart.md

```diff
@@ -0,0 +1,347 @@
+## Usage and Deployment
+
+### Prerequisites
```

Comment:
nit: Should contain minimum supported kubernetes version

Reply:
Its there on the Readme

Reply:
Do you want me to put it here as well?

Reply:
cc @niladrih